Lib/dis.py

```diff
@@ -373,6 +374,8 @@ def _get_argval_argrepr(op, arg, offset, co_consts, names, varname_from_oparg,
                 argval = offset + 2 + signed_arg*2
                 caches = _get_cache_size(_all_opname[deop])
                 argval += 2 * caches
+                if deop == ENTER_EXECUTOR:
+                    argval += 2
```

Comment:
It would be nice if we didn't need to special case this here, but that would require a change in the oparg (and implementation) or ENTER_EXECUTOR, and I figured this PR should just fix the bug in `dis`.

Reply:
Oh, this must be the cache of the underlying JUMP_BACKWARD. I think there currently is no way to get that. Not even via `_testinternalcapi`.

we should file an issue about that, it will get worse once we start putting executors on other ops.

Reply:
Created an issue here: https://github.com/python/cpython/issues/112383